ErlangInstaller/MainMenu.swift

```diff
@@ -150,11 +156,11 @@ class MainMenu: NSMenu, NSUserNotificationCenterDelegate, PopoverDelegate {
         self.statusItem = NSStatusBar.system().statusItem(withLength: NSVariableStatusItemLength)
         self.statusItem?.image = NSImage(named: "menu-bar-icon.png")
         self.statusItem?.menu = self
-		if (UserDefaults.firstLaunch) {
-			popover.contentViewController = PopoverViewController(nibName: "PopoverViewController", bundle: nil)
-			_ = Timer.scheduledTimer(timeInterval: 1.0, target: self, selector: #selector(self.showPopover(_:)), userInfo: nil, repeats: false)
-			UserDefaults.firstLaunch = false
-		}
+        if (UserDefaults.firstLaunch) {
+            popover.contentViewController = PopoverViewController(nibName: "PopoverViewController", bundle: nil)
+            _ = Timer.scheduledTimer(timeInterval: 1.0, target: self, selector: #selector(self.showPopover(_:)), userInfo: nil, repeats: false)
+            UserDefaults.firstLaunch = false
+        }
```

Comment:
I think we need to move the whole thing to spaces (specially because [it's what apple recommends](http://llvm.org/docs/CodingStandards.html#use-spaces-instead-of-tabs)), but as a separate PR.

Reply:
agreed.
I will attempt auto indent on all files before I commit